app/models/account_filter.rb

```diff
@@ -55,7 +55,7 @@ def scope_for(key, value)
     when 'by_domain'
       Account.where(domain: value.to_s.strip)
     when 'username'
-      Account.matches_username(value.to_s.strip)
+      Account.matches_username(value.to_s.strip.gsub(/^@/, ''))
```

Comment:
The following would be preferred:

```suggestion
      Account.matches_username(value.to_s.strip.delete_prefix('@'))
```